free-tunes/ExploreMusic/ExploreMusicViewModel.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  ExploreMusicViewModel.swift
+//  Free-tunes
+//
+//  Created by Tshwarelo Mafaralala on 2022/03/28.
+//
+
+import Foundation
+
+class ExploreMusicViewModel {
+    
+    private var topTracks: [Track]?
+    private weak var delegate: ExploreMusicViewModelDelegate?
+    private var repository: ExploreMusicRepository?
```

Comment:
Rather let this be a `protocol` type 😄 so that we can abstract this a little bit more.

Reply:
I agree with this 👍🏻